hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/handshakeserver/ServerProperties.java

```diff
@@ -0,0 +1,53 @@
+package ca.bc.gov.hlth.hnclientv2.handshakeserver;
+
+/**
+ * Simple POJO to store server properties in one place.
+ */
+public class ServerProperties {
+
+	private Integer serverSocket;
+	private Integer socketReadSleepTime;
+	private Integer maxSocketReadTries;
+	private Integer threadPoolSize;
+
+	public ServerProperties(Integer serverSocket, Integer socketReadSleepTime, Integer maxSocketReadTries, Integer threadPoolSize) {
+		super();
```

Comment:
Unnecessary line of code to call super().

Reply:
Eclipse code gen :). I'll remove.